### ISSUE TITLE
Application test times out when two vertical collections share a containerSelector

### ISSUE THREAD

**Issue by steveszc** [html-next/vertical-collection#320]
Minimal reproduction repo: https://github.com/steveszc/vertical-collection-shared-container-reproduction

When a route template contains two vertical-collections that share a common containerSelector (ex a scrollable `body`) a test that visits that route will timeout on the `await visit()`. The vertical-collections work as expected when using the app, however.

**Comment by steveszc** [comment]
Maybe related: https://github.com/html-next/ember-raf-scheduler/issues/5

**Comment by steveszc** [comment]
Related: https://github.com/html-next/vertical-collection/issues/197

**Comment by steveszc** [comment]
I spent some time debugging with some teammates. We were able to identify the root cause of this issue, which is a faulty conditional inside the scroll handler util. 
TL;DR; If multiple scroll handlers are registered to the same element AND passive scroll listeners are supported, then any time after the first scroll handler is added the addScrollHandler method would begin polling (even though it already added a passive scroll listener). This extra polling would trigger an extra job to be scheduled with ember-raf-scheduler, but this extra job was unexpected/accidental and was never flushed, which caused the ember-raf-scheduler test waiter to get stuck and never resolve.  
https://github.com/html-next/vertical-collection/blob/master/addon/-private/data-view/utils/scroll-handler.js#L53 

I'll plan to submit a PR with the fix and a test in the next week or two.

**Comment by MariannaAtPlay** [comment]
Hello!
Are there any plans to fix this issue? We are running into this as well!
Thank you @steveszc for figuring out the root cause.
